### DISCUSSION TITLE
如何彻底停止seatunnel组件的服务？

### DISCUSSION THREAD

**Discussion by Python-IBM** [apache/dolphinscheduler#15011] [Q&A]
在工作任务控制台不管是点击“暂停”或“停止”操作，都没法彻底中断seatunnel的服务进程，每次都只能去重启dolphinscheduler。请问有什么办法可以解决这个问题？

**Comment by dzygoon** [comment]
你找到解决办法了吗？目前我看到的方案要么找到进程杀死，要么重启seatunnel服务。

**Reply by soulgoast** [reply]
我也遇到相同的问题，目前正在想办法解决这个问题。

**Reply by dawner84** [reply]
相同问题。现在需要人工执行 ./bin/seatunnel.sh -can jobid
